compiler/lexer.nim

```diff
@@ -1249,3 +1285,14 @@ proc getIndentWidth*(fileIdx: FileIndex, inputstream: PLLStream;
     result = tok.indent
     if result > 0 or tok.tokType == tkEof: break
   closeLexer(lex)
+
+proc getPrecedence*(fileIdx: FileIndex, s: string, cache: IdentCache, config: ConfigRef): int =
```

Comment:
I don't like this at all and am thinking of a better solution.

Reply:
I have found a better way, solved ;))